platform/windows/os_windows.cpp

```diff
@@ -1896,28 +1896,26 @@ void OS_Windows::set_custom_mouse_cursor(const RES &p_cursor, CursorShape p_shap
 		Ref<Texture> texture = p_cursor;
 		Ref<Image> image = texture->get_data();
 
-		UINT image_size = 32 * 32;
+		UINT image_size = texture->get_width() * texture->get_height();
 		UINT size = sizeof(UINT) * image_size;
 
-		ERR_FAIL_COND(texture->get_width() != 32 || texture->get_height() != 32);
+		ERR_FAIL_COND(texture->get_width() > 256 || texture->get_height() > 256);
 
 		// Create the BITMAP with alpha channel
 		COLORREF *buffer = (COLORREF *)malloc(sizeof(COLORREF) * image_size);
 
 		image->lock();
 		for (UINT index = 0; index < image_size; index++) {
-			int column_index = floor(index / 32);
-			int row_index = index % 32;
+			int column_index = floor(index / texture->get_width());
+			int row_index = index % texture->get_width();
```

Comment:
Shouldn't it be `get_height()` for the row index?

Reply:
No, this is exactly what I want.

Reply:
Because the index is a range of [0..(width x height)], when this formula is applied, everything works fine.

Reply:
Then shouldn't the column index use `texture->get_height()`?

Reply:
Nevermind, it's correct :)

Reply:
Actually... `index` is the pixel position from top to bottom and then left to right, is that correct?

So let's say I have a 5x3 (WxH) sprite and index 12, it would give: `column_index == 2` and `row_index == 2`, when it should be column 4 (divided by height) and row 0 (modulo height), no?
![image](https://user-images.githubusercontent.com/4701338/38380985-aefc5112-3905-11e8-9de3-bd61bdc31cf6.png)

Reply:
I renamed row and column index.

With a image 5x3, this is the results (This result I get with an example I made and putting printf on code):
```
r=0, c=0
r=0, c=1
r=0, c=2
r=0, c=3
r=0, c=4
r=1, c=0
r=1, c=1
r=1, c=2
r=1, c=3
r=1, c=4
r=2, c=0
r=2, c=1
r=2, c=2
r=2, c=3
r=2, c=4
```

Reply:
> So let's say I have a 5x3 (WxH) sprite and index 12, it would give: column_index == 2 and row_index == 2, when it should be column 4 (divided by height) and row 0 (modulo height), no?

No, the correct is row=2 and column=2, this will be passed to ```get_pixel(x, y)``` function.

Reply:
Thanks for the clarification :)